src/applications/personalization/profile-2/tests/e2e/profile.personal-and-contact-email-address.cypress.spec.js

```diff
@@ -0,0 +1,56 @@
+import { PROFILE_PATHS } from '../../constants';
```

Comment:
Looks like this file name is based heavily on `profile.personal-and-contact-mailing-address.cypress.spec.js` which was a bad name I put on it that I should change :(

I think a better name for this file would be something like `profile.update-email-error.cypress.spec`. I'm only getting concerned about the naming since we are getting so many Cypress tests for Profile 2.0.